src/Agent/NewRelic/Agent/Core/Utilization/VendorInfo.cs

```diff
@@ -138,7 +138,7 @@ public IDictionary<string, IVendorModel> GetVendors()
 
         public IVendorModel GetAzureFunctionVendorInfo()
         {
-            if (!(_configuration.AzureFunctionModeDetected && _configuration.AzureFunctionModeEnabled))
+            if (!_configuration.AzureFunctionModeDetected)
```

Comment:
Shouldn't this still depend on the Azure Function Mode check?

Reply:
I think we want to send the Azure Function vendor info even if azure function mode isn't enabled so that we know it's an azure function. But maybe that vendor info is what lights up the Azure Function-specific UI bits?  @ashishsinghnr can you weigh in?

Reply:
Correct, that vendor info, is what causes the tags to get added, and the UI uses the tags.